packages/tldraw/src/lib/ui/components/primitives/menus/TldrawUiMenuActionItem.tsx

```diff
@@ -11,5 +11,5 @@ export function TldrawUiMenuActionItem({ actionId = '', ...rest }: TLUiMenuActio
 	const actions = useActions()
 	const action = actions[actionId]
 	if (!action) return null
-	return <TldrawUiMenuItem {...action} {...rest} />
+	return <TldrawUiMenuItem {...action} disabled={!action.enabled?.()} {...rest} />
```

Comment:
i think this logic should be:
```
const enabled = !('enabled' in action) || action.enabled()
```

feels like if `enabled()` is missing, we shouldn't be punishing previous versions of the library, to make this more backwards-compatible.

(right now, as it stands, if the function is missing, it'll disable everything)

Reply:
Not sure about this one tbh. We then might show some actions as enabled, but when you select them nothing happens. Might prefer we go the opt in route 🤷